.prettierignore

```diff
@@ -5,6 +5,7 @@ dist/
 public/
 __IconList.tsx
 rollup.icon-list.js
+build.icon-list.js
```

Comment:
Does including build.icon-list.js here stop it from minify-ing back and forth? I was having an issue with that, so if this solves that issue, I'm very excited haha.

Reply:
Yup! That's the intent.